sql-schema/233.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO `config` VALUES (798,'alert.default_copy','true','true','Copy all email alerts to default contact','alerting',0,'general',0,'0','0');
```

Comment:
This is breaking travis, you shouldn't insert the entry with a config_id set as they are auto incrementing.

See sql-schema/221.sql on how to do it.